components/components.ignore

```diff
@@ -33,3 +33,7 @@
 # 
 # Should be delivered by illumos-gate, don't enable
 /^libtecla$/d
+
+# X11 gate is not meant to be built automatically but to allow developers
+# to test fixes and updates
+/^x11-gate$/d
```

Comment:
is it really necessary? bass-o-matic builds its components list based on p5m files, not makefiles